src/cdk-experimental/popover-edit/edit-services.ts

```diff
@@ -32,7 +31,6 @@ export class EditServices {
     readonly focusTrapFactory: FocusTrapFactory,
     readonly ngZone: NgZone,
     readonly overlay: Overlay,
-    readonly positionFactory: PopoverEditPositionStrategyFactory,
```

Comment:
This would no longer be configurable/overridable then?

Reply:
That's correct, but as far as I can see nobody was overriding it anyway. I think that when we pull this out of experimental, we should revisit the public APIs and introduce injection tokens.

Reply:
Hey guys! FWIW, `angular/router` implements a similar pattern. I think it could be used here. see [here](https://github.com/angular/angular/blob/b76b3c619bc2ff9ffdd436559a7550398a810b10/packages/router/src/page_title_strategy.ts#L38)

Reply:
Ah, I'd forgotten that there was a `useFactory` option on `@Injectable`. That being said, I still don't think that we need this provider since IMO we shouldn't allow users to pass in any arbitrary positioning strategy.